SqueakyApp.Mac/AppDelegate.fs

```diff
@@ -0,0 +1,26 @@
+﻿namespace SqueakyApp.Mac
+open System
+open Foundation
+open AppKit
+open Xamarin.Forms
+open Xamarin.Forms.Platform.MacOS
+
+[<Register ("AppDelegate")>]
+type AppDelegate () =
+    inherit FormsApplicationDelegate ()
+
+    let mutable _window = new NSWindow()
```

Comment:
Change this to be

`let mutable _window : NSWindow = null`

as part of the fix to remove the extra window.